packages/orchestration/package.json

```diff
@@ -13,9 +13,12 @@
     "codegen": "scripts/fetch-chain-info.ts",
     "prepack": "tsc --build tsconfig.build.json",
     "postpack": "git clean -f '*.d.ts*' '*.tsbuildinfo'",
-    "test": "ava",
+    "test": "AGORIC_AVA_ALLOW_UNHANDLED_REJECTIONS=3 ava",
+    "test:ci": "npm run test && npm run test:expected-fail",
+    "test:expected-fail": "ava; test $? -eq 1",
```

Comment:
I was thinking we'd target specific test files for failure, maybe with a glob pattern ?

Reply:
The combination of first running `test` and then `test:expected-fail` ensures failure if either case misbehaves.

Reply:
yeah it's potentially costly for a package like `orchestration`

Reply:
I'll rework this tomorrow; I ran out of juice today.

Reply:
By using an AVA macro, I've now made the expected unhandled rejections a per-test setting, not per-file or per-package.